src/OrchardCore.Modules/OrchardCore.Templates/Views/Template/Index.cshtml

```diff
@@ -24,6 +24,8 @@
             </div>
         </div>
     </div>
+
+    <input class="form-control" name="DisplayText" placeholder="@T["Search all templates"]" type="search" />
```

Comment:
1. Ez teljes szélességű, pedig olyannak kellene lennie, mint a content item lista alatti.
2. Rálóg a textbox az alatta levő sávra.
![image](https://user-images.githubusercontent.com/1976647/98753308-51ed4b00-23c4-11eb-9970-c52526210107.png)
3. Nem ugyanaz a betűtípus, mint a content item lista alatti (vagy a List Part alatti) searchnél.
4. Nem marad meg a search box értéke.

Reply:
A szélesség és a margin javítva. A font family-t viszont ellenőriztem a placeholder és az "items" esetében is, de mindenhol ugyanolyan devTools szerint. 
Templates-nél szükséges hogy megmaradjon a search értéke? Mert azt beszéltük itt elég simán get-el a controller-nek átadni, de ha meg akarjuk tartani akkor viewmodel és displayDriver is kell gondolom.

Reply:
3. Pedig máshogy néz ki. Akkor a méret más, vagy valami egyéb.
4. Igen, és nem kell őj viewmodel. Az actionben ugye ott van a `displayText`, hozzá kell adni egy property-t a `TemplateIndexViewModel` és azt kiolvasni a cshtml-ben.

Reply:
Ami különbséget észrevettem hogy templates-nél nagyobb volt a kereső magassága, beállítottam ugyanazzal a class-al, szerintem így már egyforma.
A kereső érték megtartás sikerült.